tests/general/InvalidLinkBearTest.py

```diff
@@ -28,10 +29,12 @@ class InvalidLinkBearTest(unittest.TestCase):
     def setUp(self):
         self.section = Section("")
 
-    def assertResult(self, valid_file=None, invalid_file=None):
+    def assertResult(self, valid_file=None, invalid_file=None, settings={}):
         with requests_mock.Mocker() as m:
             InvalidLinkBear.check_prerequisites = lambda *args: True
             uut = InvalidLinkBear(self.section, Queue())
+            for name, value in settings.items():
```

Comment:
so isn't this wrong? i mean we should provide a means to test a setting only when the setting is added right? atleast a reorderin of commits is needed then in this case